doc/contributing/quick-start.chapter.md

```diff
@@ -0,0 +1,77 @@
+# Quick Start to Adding a Package {#chap-quick-start}
+
+To add a package to Nixpkgs:
+
+1. Checkout the Nixpkgs source tree:
+
+   ```ShellSession
+   $ git clone https://github.com/NixOS/nixpkgs
+   $ cd nixpkgs
+   ```
+
+2. Find a good place in the Nixpkgs tree to add the Nix expression for your package. For instance, a library package typically goes into `pkgs/development/libraries/pkgname`, while a web browser goes into `pkgs/applications/networking/browsers/pkgname`. See <xref linkend="sec-organisation" /> for some hints on the tree organisation. Create a directory for your package, e.g.
```

Comment:
Is this xref supposed to be here?

Reply:
I guess this is equivalent to `See [Section 18.3, “File naming and organisation”](#sec-organisation) for some hints...`, by using `xref` we can always get the right section title, other files like [`multiple-output.chapter.md`](https://raw.githubusercontent.com/NixOS/nixpkgs/master/doc/stdenv/multiple-output.chapter.md) and [`stdenv.chapter.md`](https://raw.githubusercontent.com/NixOS/nixpkgs/master/doc/stdenv/stdenv.chapter.md) also use `xref`.